File: app/src/main/java/ai/elimu/content_provider/ui/sound/SoundsViewModel.java
package ai.elimu.content_provider.ui.sound;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SoundsViewModel extends ViewModel {

    private MutableLiveData<String> text;

    public SoundsViewModel() {
        text = new MutableLiveData<>();
        text.setValue("SoundsViewModel");
    }

    public LiveData<String> getText() {
        return text;
    }
}

File: app/src/main/java/ai/elimu/content_provider/ui/sound/SoundsViewModel.kt
package ai.elimu.content_provider.ui.sound

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class SoundsViewModel : ViewModel() {
    private val text = MutableLiveData<String>()

    init {
        text.value = "SoundsViewModel"
    }

    fun getText(): LiveData<String> {
        return text
    }
}